pynamodb/connection/signals.py

```diff
@@ -0,0 +1,44 @@
+"""
+Implements signals based on blinker if available, otherwise
+falls silently back to a noop.
+"""
+signals_available = False
+try:
+    from blinker import Namespace
+
+    signals_available = True
+except ImportError:
+    class Namespace(object):
+        def signal(self, name, doc=None):
+            return _FakeSignal(name, doc)
+
+
+    class _FakeSignal(object):
```

Comment:
I made installing blinker optional, as this is what Flask and MongoEngine do as well.